src/mon/MgrMap.h

```diff
@@ -490,7 +490,18 @@ class MgrMap
     // One or the other, not both
     ceph_assert((ss != nullptr) != (f != nullptr));
     if (f) {
-      dump(f);
+      f->dump_bool("available", available);
```

Comment:
```
2019-08-08T00:36:08.588 INFO:tasks.cephfs_test_runner:ERROR: setUpClass (tasks.mgr.test_module_selftest.TestModuleSelftest)
2019-08-08T00:36:08.588 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2019-08-08T00:36:08.588 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2019-08-08T00:36:08.588 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-06-1102/qa/tasks/mgr/mgr_test_case.py", line 106, in setUpClass
2019-08-08T00:36:08.588 INFO:tasks.cephfs_test_runner:    cls.setup_mgrs()
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-06-1102/qa/tasks/mgr/mgr_test_case.py", line 88, in setup_mgrs
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:    timeout=20)
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-06-1102/qa/tasks/ceph_test_case.py", line 158, in wait_until_true
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:    if condition():
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-06-1102/qa/tasks/mgr/mgr_test_case.py", line 87, in <lambda>
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:    cls.wait_until_true(lambda: cls.mgr_cluster.get_active_id() != "",
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2019-08-06-1102/qa/tasks/mgr/mgr_test_case.py", line 43, in get_active_id
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:    return self.get_mgr_map()["active_name"]
2019-08-08T00:36:08.589 INFO:tasks.cephfs_test_runner:KeyError: 'active_name'
```
this literally removes "active_name" from the mgrmap exposed for mgr modules.

see http://pulpito.ceph.com/kchai-2019-08-07_23:40:20-rados-wip-kefu-testing-2019-08-06-1102-distro-basic-mira/4196350/